tests/core/failover/task.yaml

```diff
@@ -109,7 +122,7 @@ execute: |
         fi
 
         # check boot env vars
-        readlink /snap/core/current > failBoot
+        readlink "/snap/$core_name/current" > failBoot
```

Comment:
Do we want `$core_name` or `$TARGET_SNAP_NAME` here?

Reply:
Good point $TARGET_SNAP_NAME, but it works by accident since there was only ever running for the core* snaps. I'm working on making it work for the kernel snap too, so I'll fix it there.